tests/rules/PdoStatementExecuteMethodRuleTest.php

```diff
@@ -114,6 +114,28 @@ public function testParameterErrors(): void
                 'Query expects placeholder :adaid, but it is missing from values given.',
                 73,
             ],
+            /* https://github.com/staabm/phpstan-dba/issues/422
+            [
+                'Query expects 3 placeholders, but no values are given.',
+                110,
+            ],
+            [
+                'Query expects placeholder :placeholder, but it is missing from values given.',
+                116,
+            ],
+            [
+                'Query expects placeholder :can, but it is missing from values given.',
+                122,
+            ],
+            [
+                'Query expects placeholder :more, but it is missing from values given.',
+                122,
+            ],
+            [
+                'Query expects placeholder :placeholders, but it is missing from values given.',
+                122,
+            ],
+            */
```

Comment:
why?

Reply:
Those are the errors reported before implementing fix. They're no longer valid, but leaving them commented as per this files style (I think, there are already commented out ones)

Reply:
ok, make sense :+1:

Reply:
please drop these commented lines. 

the similar stuff in line 18 was commented, as I was not sure about the tests and removing the test-lines would have the consequence that all error expectation had to be adjusted (because removing lines, makes all reported error lines change)